Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -146,7 +142,10 @@ theorem omega_le_oadd (e n a) : ω ^ repr e ≤ repr (oadd e n a) := by
 theorem oadd_pos (e n a) : 0 < oadd e n a :=
   @lt_of_lt_of_le _ _ _ (ω ^ repr e) _ (opow_pos (repr e) omega_pos) (omega_le_oadd e n a)
 
-/-- Compare ordinal notations -/
+/-- Compare ordinal notations.
```

Comment:
Likewise. You *could* do "Notation for comparing ordinals: the rest of your docstring"; that seems *even* nicer to me.